Xamarin.Forms.Platform.iOS/Renderers/EditorRenderer.cs

```diff
@@ -136,6 +138,8 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateKeyboard();
 			else if (e.PropertyName == VisualElement.IsEnabledProperty.PropertyName)
 				UpdateEditable();
+			else if (e.PropertyName == VisualElement.IsEnabledProperty.PropertyName || e.PropertyName == Xamarin.Forms.InputView.IsReadOnlyProperty.PropertyName)
+				UpdateUserInteraction();
```

Comment:
Should this be deleted
```C#
			else if (e.PropertyName == VisualElement.IsEnabledProperty.PropertyName)
				UpdateEditable();
```

Since this is an else the first part of this will never happen right? since the operation above it swallows it

```C#
else if (e.PropertyName == VisualElement.IsEnabledProperty.PropertyName || e.PropertyName == Xamarin.Forms.InputView.IsReadOnlyProperty.PropertyName)
				UpdateUserInteraction();
```

And UpdateUserInteraction calls UpdateEditable